### PR TITLE
README updated: 

### PR DESCRIPTION
**Pull request by ashharr**
Solves #415 

## PR Checklist

These are the things you need to ensure are covered in your PR, otherwise the CODEOWNERS will not approve your PR, not matter how much you ping them to do so on the Discord  
  
 1. The description in the PR is a fair representation of what the PR is about.
 1. The PR is fixing one thing and one thing only.
 1. Logging. if you have N variables initialzed/used in your function, you should print all of them out to the log using logging module at least once or have a good reason why you arent.
 1. If your PR is doing something like adding a new line or removing a new line, CODEOWNERS reserve the right to ask that you undo that change unless it was for a specific reason.
 1. If you are adding a new command....document. Document the following things on help.json and the README.md
    1. The purpose of the command
    1. If the argument is called with any arguments.
       1. If it is called with any arguments, please either provide a good enough explanation of the arg that a user can tell what it will do before using the command. adding an example of how to call it with the args is not necessary but good practice.
 1. If you are making a new Class of commands, add the class to bot.json following the convention already there.
 1. Evidence of Testing. This one needs to be completed after the PR is opened. At that point, you will go on the channel on the CSSS Wall-E Test Server that was automatically created when the PR was opened and then test the [following functionality](https://github.com/CSSS/wall_e/blob/master/documentation/Working_on_Bot/Test_Cases.md). Once you had done so, you can leave a comment on the PR stating that you had done the necessary testing.
 1. Please provide ways to test whatever you just modified on the bot in the [Test Cases section](https://github.com/CSSS/wall_e/blob/master/documentation/Working_on_Bot/Test_Cases.md) so that future PRs can be tested to ensure they dont break your code when merging to master

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,8 @@
 Wall-E, named after the lovable character [Wall-E](https://en.wikipedia.org/wiki/WALL-E), is the CSSS Discord Bot. This bot is owned by the CSSS and will be maintained by the Bot_manager team.  
 
 What commands do we have? You can find this out by looking at the file located at the following directory [(`commands_to_load/help.json`)](https://github.com/CSSS/wall_e/blob/master/commands_to_load/help.json) or log onto the [discord server](https://discord.gg/9jZ5wa8?fbclid=IwAR3rKhuygMViNlOH0CoLmIloA-X-5RjIelevzIj25uwEHq-RimIb5T1xWYs) and run the command `.help`  
+
+
+### Additional Resources
```
Comment (modernNeo) [inline README.md:14]: this README is not for dev work instructions, dev work should be in the `How to work on the bot` README

**Review by modernNeo — CHANGES_REQUESTED** [review]

**Comment by ashharr** [comment]
removed it and added it there. please review.

**Comment by modernNeo** [comment]
please clean the commit history so that you are modifying only the `How to work on the bot` README. do you know how to do that?

**Comment by ashharr** [comment]
> please clean the commit history so that you are modifying only the `How to work on the bot` README. do you know how to do that?

nope, is there a quick command to clean commit history?

**Comment by modernNeo** [comment]
are you on the CSSS discord guild?

**Comment by ashharr** [comment]
no. can you add me there, my user tag is @electrohead

**Comment by modernNeo** [comment]
> no. can you add me there, my user tag is @electrohead

sorry, you are not on the guild but I can tag you there with that username?

**Comment by modernNeo** [comment]
> no. can you add me there, my user tag is @electrohead

just join via http://discord.sfucsss.org/

**Comment by modernNeo** [comment]
PR no longer needed now that documentation has been moved to README

**Pull request closed without merging**